query/search/sort.go

```diff
@@ -0,0 +1,90 @@
+package search
+
+import (
+	"github.com/buger/jsonparser"
+	jsoniter "github.com/json-iterator/go"
+	api "github.com/tigrisdata/tigris/api/server/v1"
+)
+
+// TODO: Update this to 3 once https://github.com/typesense/typesense/issues/690 is resolved
+const maxSortOrders = 2
```

Comment:
I have raised an issue with TS, it appears that they may have fixed this in `v24`. For now limiting our implementation to a primary sort order and 1 tie-breaker only. Once, the issue is resolved, we can accept two tie-breaker ranking params